ui/src/app/utils/authentication.tsx

```diff
@@ -68,7 +69,6 @@ const makeAuth2 = (config: ConfigResponse): Promise<any> => {
             reject(authError);
           }
         );
-      resolve(gapi.auth2);
```

Comment:
This resolves the promise before the auth2 library is initialized, swallowing any error that would otherwise be reported by the `reject` above.